ui-components/src/navigation/LanguageNav.tsx

```diff
@@ -8,14 +9,15 @@ export type LangItem = {
 }
 
 export interface LanguageNavProps {
+  ariaLabel?: string
   languages: LangItem[]
 }
 
-const LanguageNav = ({ languages }: LanguageNavProps) => {
+const LanguageNav = ({ ariaLabel, languages }: LanguageNavProps) => {
   return (
     <div className="language-bar">
       <div className="language-bar__inner">
-        <nav className="language-nav">
+        <nav aria-label={ariaLabel || t("languages.choose")} className="language-nav">
```

Comment:
Love this ariaLabel addition. One note is that we shouldn't add any translation strings into ui-components since our translations aren't available to consumers. I think just the prop which allows us to pass whatever we like will do 👍